lib/src/models/submission_impl.dart

```diff
@@ -136,6 +136,14 @@ class Submission extends SubmissionRef
     return _comments;
   }
 
+
+  @override
+  Map<String, String> get infoParams {
```

Comment:
Style-wise, this can be rewritten as:

```dart
Map<String, String> get infoParams => {
  'sort': commentSortTypeToString(commentSort)
};
```

Not a big deal either way.